azure-pipelines/windows_test_jobs.yml

```diff
@@ -32,4 +32,3 @@ jobs:
         platform: windows
         pythonVersion: ${{ pyver.value }}
         windowsReactor: ${{ parameters.reactor }}
-        allowFailure: ${{ eq(pyver.value, '3.8') }}
```

Comment:
I don't see any benefit in crossing this with PR #1259 and ticket [9809](https://twistedmatrix.com/trac/ticket/9809).

Reply:
This is a minor one line change which simply removes ignoring failurs on Python 3.8
during Windows CI.  It's OK to leave this in here and address ticket 9809 at the same time.

Reply:
If there weren't an existing PR addressing the existing ticket, sure.  I don't get why it needs to be confused by mixing it in here.